packages/grafana-ui/src/components/Select/MultiValue.tsx

```diff
@@ -27,7 +28,7 @@ export const MultiValueRemove: React.FC<MultiValueRemoveProps> = ({ children, in
   const styles = getSelectStyles(theme);
   return (
     <div {...innerProps} className={styles.multiValueRemove}>
-      <Icon name="times" size="sm" />
+      <IconButton name="times" size="sm" aria-label="select multi clear value" />
```

Comment:
Actually, it looks like the `<div>` on line 30 is the actual interactable element (which receives the onClick handler), so IT should either be a `<button>`/`<IconButton>`, or have role=button. 

It already has an `aria-label`, so we shouldn't need one on the Icon.

![image](https://user-images.githubusercontent.com/46142/150925467-436afe4a-1325-478a-8c31-f094406f71b6.png)

I think you should be able to do just:


```tsx
<div {...innerProps} className={styles.multiValueValue} aria-role="button">
  <Icon name="times" size="sm" />
</div>
```

:)

Reply:
I tried it as you suggested but seems like `aria-role` cannot be used there:

![Screenshot from 2022-01-25 12-15-46](https://user-images.githubusercontent.com/4025665/150968386-0a717690-0649-4675-8c01-bf4a45278b86.png)

I have also tried replacing the `div` with a `button` so the role is implicit but that completely breaks the style:

![Screenshot from 2022-01-25 12-22-53](https://user-images.githubusercontent.com/4025665/150968504-60995c82-dc20-4625-a8c4-e918996ce42b.png)

Also, using `IconButton` without an `aria-label` (because as you said, the aria-label is already in the div) won't work either.

The only solution to avoid the duplicity of the aria-label is to set the div with `role=button` (my last commit). Let me know what you think.

Reply:
Sorry yes!!! I meant `role` and not `aria-role`!!!

This looks good 👍🏻 👍🏻

Reply:
Just using `role=button` won't make the element keyboard accessible.
Would using the following not work?
```tsx
  return <IconButton {...innerProps} name="times" size="sm" className={styles.multiValueRemove} />;
```

Reply:
Thanks for the suggestion, that works too @kaydelaney :+1: 

One minor change is that now a circle appears around the close icon but that should be fine:

![Peek 2022-01-25 16-42](https://user-images.githubusercontent.com/4025665/151009524-df7d6e5b-5706-4332-a536-11e238b22d4d.gif)